src/engine/exchange.ts

```diff
@@ -1,5 +1,5 @@
 import { ExchangeConf, StrategyConf, Base } from '@m8bTypes'
```

Comment:
This file has a compilation error for me.
There is a `{ exchange: this.exchangeName, symbol } ` missing in line 48 and 66.

As hint: if you try a yarn rebuild before commiting you will find those errors.

Reply:
Good catch. Was relying too much on the tests. 👍